recipes/websocketpp/all/conanfile.py

```diff
@@ -41,11 +41,11 @@ def package(self):
         self._patch_sources()
         self.copy(pattern="COPYING", dst="licenses", src=self._source_subfolder)
         # We have to copy the headers manually, since the current cmake.install() step in the 0.8.1 release doesn't do so.
-        self.copy(pattern="*.hpp", dst="include/websocketpp", src=self._source_subfolder + '/websocketpp')
+        self.copy(pattern="*.hpp", dst="include/websocketpp", src=self._source_subfolder + "/websocketpp")
 
-    def package_info(self):    
-        if self.options.asio == 'standalone':
-            self.cpp_info.defines.extend(['ASIO_STANDALONE', '_WEBSOCKETPP_CPP11_STL_'])
+    def package_info(self):
+        if self.options.asio == "standalone":
+            self.cpp_info.defines.extend(["ASIO_STANDALONE", "_WEBSOCKETPP_CPP11_STL_"])
```

Comment:
it's not a big deal, but why are we changing string literals? are there any (dis)advantages?

Reply:
no, just convention. As we are C++ developers, maybe we don't see `'` for string, only `"`

Reply:
yeah, but as it's not a C++, it's pointless to change that

Reply:
Yes, I agree.

Reply:
me three. the usage was inconsistent :man_shrugging:

Reply:
np, leave as is :)

Reply:
Both quotes are fine, just do not add a PR only changing quotes (it will create a useless recipe revision) 🤷